config/am-scripts/scripts-content/ch-fetch-company-info.js

```diff
@@ -170,7 +172,7 @@ function fetchCompany(idmToken, companyNumber) {
                         fieldName: "IDToken2",
                         anchor: "IDToken2"
                     }],
-                    'companyNumber': companyNumber
+                    'companyNumber': { number: companyNumber }
```

Comment:
Minor error here, should be  `"company": {number: companyNumber}`, same as line 190